src/services/usersService.js

```diff
@@ -0,0 +1,38 @@
+const { models } = require('../models/models.js');
+const { User } = models;
+
+const getAllUsers = async () => {
+  const result = await User.findAll();
+
+  return result;
+};
+
+const getUserById = async (id) => {
+  return User.findByPk(id);
+};
+
+const createUser = (name) => {
+  return User.create({ name });
+};
+
+const deleteUser = async (id) => {
+  await User.destroy({
+    where: {
+      id,
+    },
+  });
+};
+
+const updateUser = async ({ id, name }) => {
```

Comment:
Ensure that the `name` parameter is validated before updating the user to prevent invalid data from being saved.